lib/administration/user/CMakeLists.txt

```diff
@@ -17,6 +18,7 @@ target_include_directories(
   PUBLIC
   "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/administration/zone/include>"
```

Comment:
This was added because of the externalization of the `zone_type` enumeration.

While this works, I wonder if this is the correct cmake change for this.

Reply:
This might need to be propagated from further up. I'll take a look later

Reply:
Sure thing.

Reply:
This seems fine to me